### DISCUSSION TITLE
Unusually large database backup size

### DISCUSSION THREAD

**Discussion by umaar** [planetscale/discussion#424] [Bug Reports] [answered]
Hey, maybe this is expected but I'm seeing my tiny testing database, as shown in the `/backups` page, being over 140 MB. The strange part is that the database has one table with one row, and a tiny schema. Why could this be? Any does that size eat into the `5 GB storage` on the hobby plan?

**Comment by jgreet** [comment, marked as answer]
Hi @umaar 

Backups contain the entire database including the system tables, not just your data. Backups for new databases start at around 140MB and increase as your tables grow. 

Storage for backups does not count against your database plan limit.
